Replicated/Replicated.cpp

```diff
@@ -4,7 +4,10 @@
 
 #include "Replicated.h"
 
-
+const Vector4 Replicated::PLATINUM = Vector4(0.941f, 0.918f, 0.839f, 1.0f);
```

Comment:
Small thing but use constexpr with vector classes they support it, means it's all done at compile time.

Reply:
![7lud16](https://github.com/The-Newcastle-Gang/SpeedFun/assets/122457074/58db1c6a-6ad6-485d-b099-3d36deb6fd93)